ethcontract-derive/src/lib.rs

```diff
@@ -342,6 +342,7 @@ impl Parse for Method {
                 })
                 .collect::<ParseResult<Vec<_>>>()?;
 
+            #[allow(deprecated)]
```

Comment:
Needed because we have to create an instance of the struct which involves touching the deprecated `constant` field.

Reply:
Tried that but doesn't work.